skywalking/trace/context/__init__.py

```diff
@@ -118,23 +119,44 @@ def active_span(self):
 
         return None
 
+    def get_correlation(self, key):
+        if key in self._correlation:
+            return self._correlation[key]
+        return None
+
+    def put_correlation(self, key, value):
+        if key is None or value is None:
```

Comment:
If the `value is None`, I'm expecting the users want to remove the item, but you just ignore it

Reply:
ok